edit.html

```diff
@@ -17,6 +17,7 @@
     <script src="content/apply.js"></script>
 
     <script src="js/sections-util.js"></script>
+    <script src="js/event-emitter.js"></script>
```

Comment:
event-emitter.js should be moved inside base.js.

The reset of these added `script` and `link` should be removed from html, and they should be loaded on demand using `require` after the user clicks button to show the settings in a modal.

Reply:
Until we have a build system it's important to keep the amount of files loaded synchronously to a bare minimum that's necessary to show the editor.